### DISCUSSION TITLE
更新1.1.17.1262后，腾讯电脑管家检测copilot为木马

### DISCUSSION THREAD

**Discussion by f1426912008** [community/community#15559] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/76981051/165066262-5e8f898b-aaf5-4a00-9675-8c7146e5bbd3.png)

更新后，代码提示无法使用

**Comment by iPisces77** [comment]
你这个也太离谱了

**Reply by nyngwang** [reply]
jo lee por

**Comment by jansorg** [comment, marked as answer]
Please try to update to the latest release, 1.1.19, which has a fix for a bug, which could be responsible for this problem.
